### PR TITLE
Rotate xticks of summary plots

### PR DESCRIPTION
**Pull request by energyLS**
## Changes proposed in this Pull Request
The xticks in the `plot_summary` functions were not rotated. When plotting many scenarios with long names, they have overlapped. Here, a rotation of 45 degrees of the labels is introduced.
Also, the link width is increased to show hydrogen pipelines with low capacity.

## Checklist

- [x] I tested my contribution locally and it seems to work fine.
- [x] Code and workflow changes are sufficiently documented.
- [x] Newly introduced dependencies are added to `envs/environment.yaml` and `envs/environment.docs.yaml`.
- [x] Changes in configuration options are added in all of `config.default.yaml`, `config.tutorial.yaml`, and `test/config.test1.yaml`.
- [x] Changes in configuration options are also documented in `doc/configtables/*.csv` and line references are adjusted in `doc/configuration.rst` and `doc/tutorial.rst`.
- [x] A note for the release notes `doc/release_notes.rst` is amended in the format of previous release notes, including reference to the requested PR.

### CODE REVIEW

**Comment by energyLS** [comment]
Hey @Eddy-JV I have requested your review here, I think it is a pretty easy and short one so nice to start with! :) Please add an "approving" review type. If you have any questions, just let me know :)

File: scripts/plot_summary.py [hunk 1]
```diff
@@ -145,7 +145,10 @@ def plot_costs():
         df.index.difference(preferred_order)
     )
 
-    new_columns = df.sum().sort_values().index
+    # Sort columns by sum
+    new_columns = (
+        df.columns
+    )  # If you want to sort by values, then use: new_columns = df.columns.sort_values()
```
Comment (Eddy-JV) [inline scripts/plot_summary.py:151]: Shouldn't this be:
` )  # If you want to sort by values, then use: new_columns = df.sum().sort_values().index`

Comment (energyLS) [reply scripts/plot_summary.py:151]: True! I will change it

Comment (energyLS) [inline scripts/plot_summary.py:151]:
```suggestion
    )  # If you want to sort by values, then use: new_columns = df.sum().sort_values().index
```

Comment (Eddy-JV) [reply scripts/plot_summary.py]: Shouldn't also the line above it as: new_columns = df.sum().index
I mean only remove the sorting.  You wrote instead: new_columns = (df.columns)

Comment (energyLS) [reply scripts/plot_summary.py]: > Shouldn't also the line above it as: new_columns = df.sum().index I mean only remove the sorting. You wrote instead: new_columns = (df.columns)

This is not required in this case and works as intended

**Review by hazemakhalek — APPROVED** [review]

**Pull request merged**